### PR TITLE
feat: adds checkmark on intent success

### PR DESCRIPTION
**Pull request by gregorylegarec**
**⚠️  https://github.com/cozy/cozy-onboarding-v3/pull/26 must be merged first**

### CODE REVIEW

File: src/assets/sprites/icon-checked-16.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```
Comment (CPatchane) [inline src/assets/sprites/icon-checked-16.svg]: In order to be more coherent, we don't need to specify the icon size in the file name here, just `icon-checked.svg` should be enough.

File: src/styles/app.styl [hunk 2]
```diff
@@ -522,6 +529,35 @@ footer
...
+    .icon-check
+      position            absolute
+      top                 (-2/base-font-size)em
+      right               (8/base-font-size)em
+      width               (24/base-font-size)em
+      height              (24/base-font-size)em
+      border-radius       (12/base-font-size)em
+      background-repeat   no-repeat
+      background-color    #35CE68
+      background-image    embedurl("../../assets/sprites/icon-checked-16.svg");
```
Comment (CPatchane) [inline src/styles/app.styl]: According to my previous comment, don't forget to change the icon file name here too.

File: src/styles/app.styl [hunk 3]
```diff
@@ -522,6 +529,35 @@ footer
       &[aria-busy=true]::after
         @extend $icon-spinner-blue
 
+.service-success
+    display          none
+    justify-content space-around
```
Comment (CPatchane) [inline src/styles/app.styl]: These two properties values could be aligned with others.

**Review by CPatchane — APPROVED** [review]
Seems great 👍, I just have two very little remarks.

**Comment by CPatchane** [comment]
Again here, this PR need to be rebased before merging.

**Review by doubleface — COMMENTED** [review]
I read it but could not point anything wrong.

**Pull request merged**